src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -119,6 +131,7 @@ export function MessagesList() {
         // really large - and the normal chat behavior would be to still scroll to the end if it's only one
         // message - we ignore this rule if there's only one additional message
         if (
+          !keyboardIsOpening.value &&
```

Comment:
We let the animated reaction handle this now, so never scroll on content size changes due to resizing of the container.